### ISSUE TITLE
Ability to use $TO {state} as KEY in afterTransitionHooks()

### ISSUE THREAD

**Issue by dclindner** [asantibanez/laravel-eloquent-state-machines#17]
### Summary
The ability to use the $TO state as the Key in afterTransitionHooks(), or equivalent function.  When you only care about the $to state, and don't care about which state it came from, it is cumbersome to use the existing method where the $from state is the key to the array.

### Proposal
Several options exist.  

Option 1 would be to nest an additional array in return of afterTransitionHook.  One array with $from state as key, and one array with $to state as key.  For example:

```
public function afterTransitionHooks(): array
{
    return [
        'from' => [
        ],
        'to' => [
        ]
    ];
}
```

Option 2 is to implement another function for $to states.

Option 3 would be to accept a return object that contains two arrays/collections keyed $to and $from, respectively.

I'm sure there are many others, too, likely others better.

### Additional Notes
Reference links or comments

**Comment by asantibanez** [comment]
Hi @dclindner 

Thanks for your interest in the package.

Initially, the `afterTransitionHooks` worked with `$to` state for keys. I might revert that behavior since your point is valid and makes sense.

I'll add this during the following week. 👍

Thanks again!

**Comment by dclindner** [comment]
Thank you.  I look forward to it.  :-)

**Comment by asantibanez** [comment]
@dclindner available now in v5.0.0 https://github.com/asantibanez/laravel-eloquent-state-machines/releases/tag/v5.0.0

**Comment by dclindner** [comment]
Thank you, very much!
